packages/pacget-git/pacget-git.pacscript

```diff
@@ -9,33 +9,35 @@ url="https://gitlab.com/Erik.Lundstedt/pacget.git"
 
 license="Don't Be a Jerk"
 depends="fzf curl bash"
+gives="$pkgname"
 description="A fully foss, lightweight fuzzysearching TUI frontend for pacstall using fzf"
 removescript="yes"
 
 debian="TRUE"
 ubuntu="TRUE"
 
 pkgver() {
-    git ls-remote "${url}" master | cut -f1
+	git ls-remote "${url}" main | cut -f1
 }
 version="$(pkgver)"
 
 ## installation steps
 #preparations
 prepare() {
-    true
+	true
 }
 # pacget is a bashscript, no need to build anything
 build() {
     true
 }
-#this just coppies the files over and makes the file executable
+#this just copies the files over and makes the file executable
 install() {
-    sudo install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
-    sudo install -Dm755 "pacget" -t "${pkgdir}/usr/bin/"
-    sudo install -Dm755 "pacget_install" -t "${pkgdir}/usr/bin/"
+	install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+    install -Dm755 "pacget" "/usr/src/$name/pacget/usr/bin/pacget"
+
 }
 
 removescript() {
-    sudo rm -rf "/usr/share/licenses/${pkgname}"
+	rm -rf /usr/src/pacstall/pacget
+	rm -rf "/usr/share/licenses/${pkgname}"
```

Comment:
Why no `sudo`?

Reply:
https://discord.com/channels/839818021207801878/848159751161774100/880796283458187305

Reply:
@Henryws says
> so, fakeroot is being removed because our original intent was that this would mitigate sudo and make it harder to tamper with the system. Fakeroot doesn’t do that, after further digging, and sudo was/is able to be run inside fakeroot (which is why we added it: to prevent root access). Fakeroot also is +1 dep for pacstall, and is, at this point, just a burden to continue to support

Reply:
i readded sudo